web/src/components/RateScene/RateScene.tsx

```diff
@@ -29,14 +29,30 @@ const RateScene = ({
   sceneId,
   previous,
   onRateSuccess,
+  currentUserRating,
 }: {
   sceneId: string
   previous: () => void
   onRateSuccess: () => void
+  currentUserRating?: {
+    id: string
+    live: boolean
+    vibe: number
+    crowded: number
+  }
 }) => {
   const [live, setLive] = useState<boolean | null>(null)
-  const [vibe, setVibe] = useState({ x: 5, y: 0 })
-  const [crowded, setCrowded] = useState({ x: 5, y: 0 })
+  const [vibe, setVibe] = useState({
+    x: typeof currentUserRating?.vibe === 'number' ? currentUserRating.vibe : 5,
```

Comment:
Can also `Number(currentUserRating?.vibe) || 5`, let me know if that works.

Reply:
Hmm...for some reason that is not working so I will leave it as is and merge